lib/xebow/animation.ex

```diff
@@ -4,11 +4,12 @@ defmodule Xebow.Animation do
 
   There are currently two distinct ways to define an animation.
 
-  You may define an animation with a predefined `:frames` field. Each frame will advance every `:delay_ms` milliseconds. These animations should use the `Xebow.Animation.Loop` `:type`. See the moduledoc of `Xebow.Animation.Loop` for an example. These animations should have a nil `:frame_buffer_length`.
+  You may define an animation with a predefined `:frames` field. Each frame will advance every `:delay_ms` milliseconds.
+  These animations should use the `Xebow.Animation.{OneShot,Loop}` `:type`. See the moduledocs of those modules for
```

Comment:
Sorry, small nitpick: It looks like `{OneShot,Loop}` is missing a space after the comma.

Reply:
Good catch, this should actually be `Xebow.Animation.Static`, fixed.